### PR TITLE
Make: Ignore the 'stringop-overflow' compiler warning

### PR DESCRIPTION
**Pull request by oleorhagen**
This is out of our control, and the warning is annoying.

Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ CGO_ENABLED=1
 export CGO_ENABLED
 
 # Get rid of useless warning in lmdb
```
Comment (lluiscampos) [inline Makefile:21]: Please update the comment too

**Review by lluiscampos — COMMENTED** [review]
Thanks for doing this! It annoys me too :)

**Review by kacf — APPROVED** [review]
:+1:

File: Makefile (hunk 1 above)
Comment (oleorhagen) [reply Makefile:21]: The warning is still in `lmdb`, so it's still right, but I can remove it still if you like

Comment (lluiscampos) [reply Makefile:21]: No, if that is the case it is fine. I thought it came from somewhere else.

**Comment by oleorhagen** [comment]
@lluiscampos will you merge this?

**Pull request merged**